### ISSUE TITLE
fix podcast player

### ISSUE THREAD

**Issue by mcfiredrill** [datafruits/datafruits.old#5]

**Comment by nikolas** [comment]
Is this referring to the play buttons here not matching up with the actual mp3 files? http://www.datafruits.fm/podcasts

**Comment by mcfiredrill** [comment]
yeah, the play button plays the right podcast, sometimes, but if you hit pause and play again the normal stream will start playing again. needs a real overhaul i think.
